src/styles/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  padding: 0 16px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url('../images/star.svg');
+    background-size: contain;
+  }
+
+  &__reviews-text {
+    padding-left: 17px;
+    font-size: 10px;
+    color: $card-title-color;
```

Comment:
Ensure that the color variables are defined and used consistently throughout the project. If $card-title-color is not defined in 'variables.scss', you should add it there to maintain consistency and ease of maintenance.